lib/validate.js

```diff
@@ -70,15 +70,19 @@ export function advPostSchemaMembers (client) {
         },
         message: `must be divisble be ${BOOST_MIN}`
       }),
-    forward: string()
-      .test({
-        name: 'name',
-        test: async name => {
-          if (!name || !name.length) return true
-          return await usernameExists(client, name)
-        },
-        message: 'stacker does not exist'
-      })
+    forward: array()
+      .max(MAX_FORWARDS)
+      .of(object().shape({
+        nym: string().required().test({
+          name: 'name',
+          test: async name => {
+            if (!name || !name.length) return true
+            return await usernameExists(client, name)
+          },
+          message: 'stacker does not exist'
+        }),
+        pct: number().required().min(1).max(100).integer()
```

Comment:
I guess I saw this comment: 
> // not sure how to use this on server ...
which made me think this validation did not execute server-side? If it does, then yea we can consolidate. I'll explore further.

Reply:
Oh lol hmmmmm. That might be an old comment because we use `advPostSchemaMembers` in `discussionSchema` (and others) which does do server side validation.

Reply:
lol I think the original sin was me leaving the comment there